### PR TITLE
Upgraded dependencies

### PR DESCRIPTION
**Pull request by nadav-fima**
Fixes this issue
https://github.com/rushio-consulting/flutter_camera_ml_vision/issues/126

### CODE REVIEW

**Comment by kevin-sakemaer** [comment]
LGTM cc @jaumard

**Review by kevin-sakemaer — DISMISSED** [review]

**Review by jaumard — DISMISSED** [review]
LGTM too

**Review by kevin-sakemaer — APPROVED** [review]
Seems good @jaumard could you have a look and test it with your project ?

File: lib/flutter_camera_ml_vision.dart [hunk 1]
```diff
@@ -110,7 +110,7 @@ class CameraMlVisionState<T> extends State<CameraMlVision<T>>
       _lastImage = '${tempDir.path}/${DateTime.now().millisecondsSinceEpoch}';
       try {
         await _cameraController.initialize();
-        await _cameraController.takePicture(_lastImage);
+        await _cameraController.takePicture();
```
Comment (jaumard) [inline lib/flutter_camera_ml_vision.dart:113]: if `_lastImage ` is not used anymore it break the pause/stop functionality who show the last frame of the video.

**Comment by jaumard** [comment]
Closing as it has been done on another PR, thanks anyway!

**Pull request closed without merging**